cmd/gitops-server/cmd/cmd.go

```diff
@@ -0,0 +1,328 @@
+package ui
+
+import (
+  "context"
+  "embed"
+  "fmt"
+  "io/fs"
+  "net"
+  "net/http"
+  "net/url"
+  "os"
+  "os/signal"
+  "path/filepath"
+  "syscall"
+  "time"
+
+  "github.com/go-logr/zapr"
+  "github.com/sirupsen/logrus"
+  "github.com/spf13/cobra"
+  "go.uber.org/zap"
+
+  "github.com/weaveworks/weave-gitops/cmd/gitops/cmderrors"
+  "github.com/weaveworks/weave-gitops/pkg/helm/watcher"
+  "github.com/weaveworks/weave-gitops/pkg/helm/watcher/cache"
+  "github.com/weaveworks/weave-gitops/pkg/kube"
+  "github.com/weaveworks/weave-gitops/pkg/server"
+  "github.com/weaveworks/weave-gitops/pkg/server/auth"
+)
+
+// Options contains all the options for the `ui run` command.
```

Comment:
nit: no ui run here, the same reference to ui run command repeats in other places

Reply:
Ah good catch, I'll go sort these, thanks!

Reply:
Think [0fc3639](https://github.com/weaveworks/weave-gitops/pull/1469/commits/0fc36393a61c93e8e9e63ad4fc9df7c8ecc1b245) should fix these